docs(configuration): add example for loader option (#3927)
webpack/webpack.js.org: src/content/configuration/other-options.md

`object`

Expose custom values into the [loader context](/api/loaders/#the-loader-context).

For example, you can define a new variable in loader context:

__webpack.config.js__

```javascript
module.exports = {
  // ...
  loader: {
    answer: 42
  }
};
```

Then use `this.answer` to get its value in loader:

> 

Reviewer: Maybe prefix it with `custom-loader.js` or smth like that for more clarity?

```javascript
module.exports = function (source) {
  // ...
  console.log(this.answer); // will log `42` here
  return source;
};
```

> T> You can override any properties in loader context as webpack copies all properties defined in `loader` to the loader context.

Reviewer: some grammar suggestion:
```suggestion
T> You can override properties in the loader context as webpack copies all properties that are defined in the `loader` to the loader context.
```

`object`

Expose custom values into the [loader context](/api/loaders/#the-loader-context).

> For example, you can define a new variable in loader context:

Reviewer: ```suggestion
For example, you can define a new variable in the loader context:
```

__webpack.config.js__

```javascript
module.exports = {
  // ...
  loader: {
    answer: 42
  }
};
```

> Then use `this.answer` to get its value in loader:

Reviewer: ```suggestion
Then use `this.answer` to get its value in the loader:
```
